src/client/Microsoft.Identity.Client/OAuth2/OAuth2Client.cs

```diff
@@ -57,6 +57,11 @@ public void AddBodyParameter(string key, string value)
             }
         }
 
+        internal void AddHttpTelemetryToHeaders(string key, string value)
```

Comment:
Looks general purpose, maybe rename to `AddHeader`

Reply:
done. htanks

---
In reply to: [389744569](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1693#discussion_r389744569) [](ancestors = 389744569)